packages/core/useVirtualList/index.ts

```diff
@@ -147,18 +309,12 @@ export function useVirtualList <T = any>(list: MaybeRef<T[]>, options: UseVirtua
     }
   })
 
-  const containerStyle: Partial<CSSStyleDeclaration> = { overflowY: 'auto' }
-
   return {
-    list: currentList,
+    calculateRange,
     scrollTo,
-    containerProps: {
-      ref: containerRef,
-      onScroll: () => {
-        calculateRange()
-      },
-      style: containerStyle,
-    },
+    containerStyle,
     wrapperProps,
+    currentList,
+    containerRef,
```

Comment:
The return types seems introduce a breaking change, is it intentional?

Reply:
If you look at this feature, you will notice that there are no breaking changes.
```js
export function useVirtualList <T = any>(list: MaybeRef<T[]>, options: UseVirtualListOptions): UseVirtualListReturn<T> {
  const { containerStyle, wrapperProps, scrollTo, calculateRange, currentList, containerRef } = 'itemHeight' in options
    ? useVerticalVirtualList(options, list)
    : useHorizontalVirtualList(options, list)

  return {
    list: currentList,
    scrollTo,
    containerProps: {
      ref: containerRef,
      onScroll: () => {
        calculateRange()
      },
      style: containerStyle,
    },
    wrapperProps,
  }
}
```

The original function still returns the same object as before.

It's just that the object has different values ​​depending on the direction of the list passed, but nothing has changed for the vertical sheet.